pkgs/desktops/gnome-3/core/gnome-terminal/default.nix

```diff
@@ -1,6 +1,7 @@
 { stdenv, fetchurl, pkgconfig, libxml2, gnome3, dconf, nautilus
 , gtk3, gsettings-desktop-schemas, vte, intltool, which, libuuid, vala
-, desktop-file-utils, itstool, wrapGAppsHook, hicolor-icon-theme }:
+, desktop-file-utils, itstool, wrapGAppsHook, hicolor-icon-theme
+, moveSystemdUserUnitsHook }:
```

Comment:
I think `moveSystemdUserUnitsHook` should go into stdenv for Linux targets but that would make this a mass-rebuild.

Reply:
Do we need to add that to every package that contains user services?

Reply:
The way this is implemented at the moment yes. But I would like to put `moveSystemdUserUnitsHook` in stdenv so it's automatically applied to all packages. But during the WIP period I'm avoiding changes in stdenv so that I don't have to rebuild the whole world when I'm testing 🙂